docs/Readme.md

```diff
@@ -4,7 +4,7 @@
 
 * [Installation](Installation.md)
   * [Background: Jupyter Notebooks](Background-Jupyter.md)
-  * [Docker Set-up](Using-Docker.md)
+  * [Using Virtual Environment](Using-Virtual-Environment.md) (Recommended)
```

Comment:
Do we need to say "(Recommended)"? My understanding is that this is the only path to install.

Reply:
removed "(Recommended)"